### PR TITLE
Yolo-v3 support, enhancements & bug-fixes

### PR DESCRIPTION
**Pull request by sjain-stanford**
This PR contains the following changes:

### Enhancements:
- Yolo-v3 support (thanks @jerry73204!) [[see PR](https://github.com/sjain-stanford/DW2TF/pull/1)]
  - Switch from tf-slim to native tf.layers API
  - New layers added e.g. shortcut (eltwise add), upsampling etc.
- Use *fused* batch norm
- Supports both training and inference versions of batch norm

### Bug-fixes
- Absorb bias in BN's beta as appropriate [see #3]
- Use darknet defaults for leaky ReLU alpha and BN's epsion/momentum [see #2]
- Restored correct BN --> ReLU ordering after merging @jerry73204's [PR](https://github.com/sjain-stanford/DW2TF/pull/1)

### CODE REVIEW

**Comment by jinyu121** [comment]
Thank you!

**Comment by jerry73204** [comment]
Thanks!
